runtime/kusama/src/lib.rs

```diff
@@ -1521,7 +1642,7 @@ impl Get<Perbill> for NominationPoolsMigrationV4OldPallet {
 ///
 /// This contains the combined migrations of the last 10 releases. It allows to skip runtime
 /// upgrades in case governance decides to do so. THE ORDER IS IMPORTANT.
-pub type Migrations = (migrations::Unreleased,);
+pub type Migrations = (migrations::Unreleased, migrations::UpgradeSessionKeys);
```

Comment:
Not sure whether this should be added here or directly to `migrations::Unreleased`, hopefully someone else will comment on this.

Reply:
Yeah, this needs to be added to the `Unreleased` type below.